docs/src/main/paradox/docs/fusion/architecture.md

```diff
@@ -36,16 +38,32 @@ configuration, and the manifest should be hosted somewhere Nexus Fusion can requ
 
 ## Customization
 
-You can customize the Header of Nexus Fusion by setting up the following environment variables:
+Customize the Nexus codebase appearance by setting environment variables. Defaults are provided by Nexus Fusion if not set.
+
+### Environment Variables for Customization
+
+![Nexus Header](./assets/environment-variables.png)
+
+#### Landing Page Customization
+
+- `LOGO_IMG`: HTTPS URL for the application logo. Recommended: SVG format with transparent background. Ideal size: ~35 px height, max 250 px width. @link:[Click here for an example SVG](https://github.com/BlueBrain/nexus-web/tree/main/src/shared/images/EPFL_BBP_logo.svg){ open=new }.
+- `LOGO_LINK`: HTTPS URL redirecting from the logo. Example: <https://www.epfl.ch>
+- `LANDING_VIDEO`: HTTPS URL for a landing page video. Requirements: MP4 format, H.264 codec, ~10MB, 1920×1080 resolution. @link:[Click here for an example video](https://github.com/BlueBrain/nexus-web/tree/main/src/videos/BrainRegionsNexusPage.mp4){ open=new }. Ensure that the video's main color is not too bright, as the text on top of it will be white.
+- `LANDING_POSTER_IMG`: HTTPS URL for a loading image on the landing page, displayed while the video loads. @link:[Click here for an example poster image](https://github.com/BlueBrain/nexus-web/tree/main/src/shared/images/BrainRegionsNexusPage.jpg){ open=new }. Please use a PNG or JPG image that matches the size and color of the video. Aim for the image to have the same dimensions as the video and be under 200 KB in size.
+- `MAIN_COLOR`: Main background color in hex code. Example color: #062d68
+
+#### Page Specific Customization
+
+![Example organization env](./assets/organizations-envs.png)
+![Example projects env](./assets/projects-envs.png)
+![Example studios env](./assets/studios-envs.png)
+
+- `ORGANIZATION_IMG`: Provide the HTTPS URL for an image on the organization page. @link:[Click here for an example image](https://github.com/BlueBrain/nexus-web/blob/main/src/shared/images/sscx-by-layers-v3.png){ open=new }. The recommended dimensions are approximately 1500 × 450 pixels. Please ensure that the image size is kept under 200 KB.
+- `PROJECTS_IMG`: Share the HTTPS URL for an image on the projects page. @link:[Click here for an example image](https://github.com/BlueBrain/nexus-web/blob/main/src/shared/images/hippocampus.png){ open=new }. The recommended dimensions are approximately 1500 × 450 pixels. Please ensure that the image size is kept under 200 KB.
+- `STUDIOS_IMG`: Provide the HTTPS URL for an image on the studios page. @link:[Click here for an example image](https://github.com/BlueBrain/nexus-web/blob/main/src/shared/images/neocortex.png){ open=new }. The recommended dimensions are approximately 1500 × 450 pixels. Please ensure that the image size is kept under 200 KB.
```

Comment:
Reading this it's a bit unclear to me if the 200 KB size is a recommendation or mandatory (i.e. if the size exceeds 200 KB, an error will be thrown). If it's a recommendation then maybe we could add somethign along the lines of "Pleas ensure that the image size is kept under 200 KB to provide a load time/user expierience."

Reply:
#4533